app/models/miq_server.rb

```diff
@@ -487,11 +487,24 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
-  def self.audit_managed_resources
-    total_vms   = VmOrTemplate.active.count
-    total_hosts = Host.active.count
+  def self.managed_resources
+    {
+      :vms                     => VmOrTemplate.active.count,
+      :hosts                   => Host.active.count,
+      :aggregate_physical_cpus => MiqRegion.my_region.aggregate_physical_cpus(Host.active),
+    }
+  end
 
-    totals = {"vms" => total_vms, "hosts" => total_hosts}
+  def self.unmanaged_resources
+    {
+      :vms                     => VmOrTemplate.not_active.count,
```

Comment:
This is weird because I think it considers archived templates as VMs?

Reply:
Yes it does, it did before as well.  `VmOrTemplate.not_active` is the same as [here](https://github.com/ManageIQ/manageiq/pull/19857/files#diff-588453599dbbedd9ea0d01064181082bL73) `VmOrTemplate.where(:ems_id => nil)`

Reply:
suggest we change this to `Vm.not_active.count` to more accurately reflect reality.